cmd/cstorvolumeclaim/cstorvolumeclaim.go

```diff
@@ -437,3 +442,27 @@ func randomizePoolList(list *apis.CStorPoolInstanceList) *apis.CStorPoolInstance
 
 	return res
 }
+
+// CVCWithVersionAndRefernceDetails build cvc resource with required version
+// details, volume references and status capacity
+func CVCWithVersionAndRefernceDetails(
+	claim *apis.CStorVolumeClaim,
+	volumeRef *corev1.ObjectReference,
+) (*apis.CStorVolumeClaim, error) {
+
+	cvcObj, err := cvc.BuildFrom(claim).
```

Comment:
we seem to be updating the claim that came in as input.. is that expected behavior of this fn?
We don't need to build.. we can just do update if thats the case..
we probably don't need function.. you can just do
`cvc.BuildFrom(claim).WithClaimRef(volumeRef).WithStatusPhase(apis.CStorVolumeClaimPhaseBound).WithNewVersion(version.GetVersion()).WithDependentsUpgrade()`

Reply:
Let CSI create the CVC CR with versionDetails, so that, we don't need changes here

Reply:
remove this from maya .. will be added from CSI side